.github/CODEOWNERS

```diff
@@ -576,8 +576,10 @@
 /core/templates/third-party-imports/ @nishantwrp
 /jinja_utils*.py @jameesjohn
 /.lighthouserc.js @jimbyo @jameesjohn
+/.lighthouserc-a11y.js @jimbyo
```

Comment:
Perhaps lighthouserc-accessibility.js (since otherwise this reads a bit like "ally").